src/scene/materials/standard-material.js

```diff
@@ -382,48 +367,34 @@ class StandardMaterial extends Material {
      */
     clone() {
         const clone = new StandardMaterial();
-        this._cloneInternal(clone);
 
-        for (let i = 0; i < _propsSerial.length; i++) {
-            const pname = _propsSerial[i];
-            if (this[pname] !== undefined) {
-                if (this[pname] && this[pname].copy) {
-                    if (clone[pname]) {
-                        clone[pname].copy(this[pname]);
-                    } else {
-                        clone[pname] = this[pname].clone();
-                    }
-                } else {
-                    clone[pname] = this[pname];
-                }
-            }
-        }
+        this._cloneInternal(clone);
 
-        return clone;
-    }
+        // set properties
+        Object.keys(_props).forEach((k) => {
+            clone[k] = this[k];
```

Comment:
no need to clone these?

Reply:
The property set implementation copies values for aggregate types (vec, color etc).